### ISSUE TITLE
model.py报错

### ISSUE THREAD

**Issue by xiumia** [Pual2013/CDL#1]
你好，model.py第74行这里有点混乱。
part_features 和 part_feature。一个有s一个没有s
part_imgs = part_imgs.view(batch * self.topN, 3, 224, 224)
_, _, part_features = self.pretrained_model(part_imgs.detach()) # part_features->part image embedding
part_feature = self.ca(part_feature) # part_feature??

model.py第74行，这个part_feature事先没有定义，会报错。
